src/controllers/info.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { getHostname } from "../config";
+
+export function idGet(req: Request, res: Response) {
+  res.json(buildClientIdDocument())
+}
+
+export function buildClientIdDocument(): ClientIdDocument {
+ return {
+    "@context": [
+      "https://www.w3.org/ns/solid/oidc-context.jsonld"
+    ],
+    "client_id": `${getHostname()}/info/id`,
+    "client_name": "GDS Solid proof of concept app",
+    "client_uri": `${getHostname()}`,
+    "post_logout_redirect_uris": [
+      `${getHostname()}`
+    ],
+    "redirect_uris": [
+      `${getHostname()}/login/callback`
```

Comment:
To support local development, we can include `http://localhost:3000/info/id` in the `redirect_uris` for the actual `ClientID Document` that gets published:

```suggestion
      `${getHostname()}/login/callback`,
      "http://localhost:3000/info/id"
```

Reply:
Thanks, I added this in [c96b4ae](https://github.com/alphagov/di-solid-prototype/pull/55/commits/c96b4aefd293ee7f58ec7ee3b2fd1ffbe76337b8)